### DISCUSSION TITLE
TC4 error

### DISCUSSION THREAD

**Discussion by RinaldAbuud** [artisan-roaster-scope/artisan#773] [Q&A]
ser.ARDUINO TC4temperature(): WriteFile failed (PermissionError(13, 'Access is denied.',None, 5))@line 5352

anyone can help me for this problem..?

**Comment by MAKOMO** [comment]
Did you read the [Installation Instructions](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/Installation.md) linked from the download page?

Seems you are on Linux (you forgot to tell) and your account is not in the dialout group.

**Comment by RinaldAbuud** [comment]
I'm using windows 10. at first from start it's working normally until FC and i open the airflow, but this error always happen if i open the airflow.

https://drive.google.com/file/d/1Og-X3quoewOloE2gM7j19ds8l7URza5c/view?usp=drivesdk


https://drive.google.com/file/d/1OecexqaKM4W_oLh1T0VkenmMCQPtrBcZ/view?usp=drivesdk

**Reply by MAKOMO** [reply]
Please attach documents that are relevant here directly (just by drag-and-drop to the message field on the GitHub page). What would be of interest would be your Artisan settings as artisan-settings.aset (export via menu Help >> Save Settings; rename the file to artisan-settings.txt to be able to attach it here). Also your hardware setup would be of interest. For example how you "open the airflow", is this via the TC4 using an actor?
